### PR TITLE
Expand documentation for token from Android backup

### PR DESCRIPTION
**Pull request by sgtio**
Add information about the initial setup of the device, and the steps
needed prior to creating the backup.

### CODE REVIEW

File: docs/discovery.rst [hunk 1]
```diff
@@ -174,7 +198,7 @@ to the tool with ``--password <password>``.
             MAC: 28:6C:07:XX:XX:XX
     room1
             Model: yeelink.light.color1
-            IP address: 192.168.XXX.XXX
+            \IP address: 192.168.XXX.XXX
```
Comment (rytilahti) [inline docs/discovery.rst]: Why change this?

Comment (sgtio) [reply docs/discovery.rst]: yup, this is accidental. I'll fix it!
Thanks for the feedback!

**Comment by sgtio** [comment]
I have squashed the commit with the typo fix. Please let me know if you prefer having 2 separate commits. Thanks!

**Comment by rytilahti** [comment]
It is fine, we'll simply squash & merge it. Thanks a lot for helping with the documentation!

**Pull request merged**